less/home-page.less

```diff
@@ -1 +1,70 @@
-// Home Content
\ No newline at end of file
+// Home Content
+.home {
```

Comment:
try to be consistent on whether or not you add a space after the class name (`.home {` 🆚 `.home{`)

I always add a space, but I guess you don't HAVE to 🤷‍♂️